verification/client.go

```diff
@@ -236,6 +239,18 @@ func (c *Client[_, _]) GetCertificateChain() (*GetCertificateChainResp, error) {
 	return &certs, nil
 }
 
+// DeriveChild() calls the DPE CertifyKey command.
```

Comment:
*calls the DPE DeriveChild command

Reply:
Hi @sree-revoori1  , seems the code flow is modified few days back and this tree needs significant changes to incorporate the new code. I shall commit the changes after rebasing the code. 

Thank you!